src/WorkOS.net/Services/MFA/_interfaces/EnrollFactorOptions.cs

```diff
@@ -0,0 +1,13 @@
+namespace WorkOS
+{
+    using Newtonsoft.Json;
+
+    public class EnrollFactorOptions : BaseOptions
```

Comment:
Since we don't intend for SDK consumers to construct this base class directly, could we add an `internal` constructor to prevent constructing it outside of this assembly?

```cs
public class EnrollFactorOptions : BaseOptions
{
    internal EnrollFactorOptions(string type)
    {
        Type = type;
    }

    /// <summary>
    /// The type of factor to enroll.
    /// </summary>
    [JsonProperty("type")]
    public string Type { get; }
}
```